Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -1375,6 +1375,11 @@ std::string DrawMol::getAtomSymbol(const Atom &atom,
     }
   } else if (isComplexQuery(&atom)) {
     symbol = "?";
+    if (atom.hasProp("molAtomMapNumber")) {
+      std::string map_num = "";
+      atom.getProp("molAtomMapNumber", map_num);
+      symbol += ":" + map_num;
+    }
```

Comment:
```suggestion
    std::string mapNum;
    if (atom.getPropIfPresent("molAtomMapNumber",mapNum)) {
      symbol += ":" + mapNum;
    }
```

Reply:
I have also changed the one I copied (a few lines down) for this one.